api/src/wfl/jdbc.clj

```diff
@@ -25,14 +25,18 @@
    (let [{:keys [line]} (meta &form)]
      `(let [db#         ~db
             sql-params# ~sql-params]
-        (logger/debug (str/join " " ["jdbc/query:" (format-db db#) sql-params#]) ~line)
+        (logger/debug
+         (str/join " " ["jdbc/query:" (format-db db#) (pr-str sql-params#)])
+         ~line)
```

Comment:
Thanks, I think.
Ah, this is to log collection contents instead of Java object references.

Reply:
Bingo!  Not a panacea for all of our weird logs, but an improvement.